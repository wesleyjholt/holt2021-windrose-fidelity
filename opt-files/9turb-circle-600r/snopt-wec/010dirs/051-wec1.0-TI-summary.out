
 ==============================
 S N O P T  7.7.3    (Dec 2019)
 ==============================
 opt-files/9turb-circle-600r/snopt-wec/010dirs/051-wec1.0-TI-print.out
 XXX  Keyword not recognized:         OPT-FILES/9TURB-
 opt-files/9turb-circle-600r/snopt-wec/010dirs/051-wec1.0-TI-summary.out
 XXX  Keyword not recognized:         OPT-FILES/9TURB-

 SNMEMA EXIT 100 -- finished successfully
 SNMEMA INFO 104 -- memory requirements estimated



 Nonlinear constraints      45     Linear constraints       1
 Nonlinear variables        18     Linear variables         0
 Jacobian  variables        18     Objective variables     18
 Total constraints          46     Total variables         18



 The user has defined     828   out of     828   first  derivatives


 Major Minors     Step   nCon Feasible  Optimal  MeritFunction     nS Penalty
     0     14               1 (2.7E-10) 6.2E-03 -4.5180256E+03     14           r

 Major Minors     Step   nCon Feasible  Optimal  MeritFunction     nS Penalty
     0      0               8 (2.7E-10) 6.2E-03 -4.5180256E+03     14           r

 Major Minors     Step   nCon Feasible  Optimal  MeritFunction     nS Penalty
     0      0              15 (2.7E-10) 6.2E-03 -4.5180256E+03     14           r

 SNOPTA EXIT  40 -- terminated after numerical difficulties
 SNOPTA INFO  41 -- current point cannot be improved

 Problem name                 opt prob
 No. of iterations                  14   Objective           -4.5180255811E+03
 No. of major iterations             0   Linear    obj. term  0.0000000000E+00
 Penalty parameter           0.000E+00   Nonlinear obj. term -4.5180255811E+03
 User function calls (total)        22
 No. of superbasics                 14   No. of basic nonlinears             4
 No. of degenerate steps             0   Percentage                       0.00
 Max x                      14 6.0E+02   Max pi                     46 1.0E+00
 Max Primal infeas          60 1.6E-07   Max Dual infeas             5 6.2E-03
 Nonlinear constraint violn    1.6E-07



 Solution printed on file  18

 Time for MPS input                             0.00 seconds
 Time for solving problem                       2.69 seconds
 Time for solution output                       0.00 seconds
 Time for constraint functions                  2.66 seconds
 Time for objective function                    0.00 seconds
