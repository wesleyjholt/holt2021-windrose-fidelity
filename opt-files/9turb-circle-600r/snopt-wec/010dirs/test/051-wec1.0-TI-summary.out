
 ==============================
 S N O P T  7.7.3    (Dec 2019)
 ==============================
 opt-files/9turb-circle-600r/snopt-wec/010dirs/051-wec1.0-TI-print.out
 XXX  Keyword not recognized:         OPT-FILES/9TURB-
 opt-files/9turb-circle-600r/snopt-wec/010dirs/051-wec1.0-TI-summary.out
 XXX  Keyword not recognized:         OPT-FILES/9TURB-

 SNMEMA EXIT 100 -- finished successfully
 SNMEMA INFO 104 -- memory requirements estimated



 Nonlinear constraints      45     Linear constraints       1
 Nonlinear variables        18     Linear variables         0
 Jacobian  variables        18     Objective variables     18
 Total constraints          46     Total variables         18



 The user has defined     828   out of     828   first  derivatives


 Major Minors     Step   nCon Feasible  Optimal  MeritFunction     nS Penalty
     0     13               1 (2.2E-12) 4.7E-04 -4.3348841E+03     13           r
     1      1  1.8E-09     11 (2.2E-12) 4.7E-04 -4.3348841E+03     13           r
     2      2  1.8E-10     18 (2.2E-12) 4.7E-04 -4.3348841E+03     14         s

 SNOPTA EXIT   0 -- finished successfully
 SNOPTA INFO   3 -- requested accuracy could not be achieved

 Problem name                 opt prob
 No. of iterations                  16   Objective           -4.3348840922E+03
 No. of major iterations             2   Linear    obj. term  0.0000000000E+00
 Penalty parameter           0.000E+00   Nonlinear obj. term -4.3348840922E+03
 User function calls (total)        43
 No. of superbasics                 14   No. of basic nonlinears             4
 No. of degenerate steps             0   Percentage                       0.00
 Max x                       6 5.9E+02   Max pi                     40 1.3E+00
 Max Primal infeas          61 1.3E-09   Max Dual infeas             8 6.2E-04
 Nonlinear constraint violn    1.3E-09



 Solution printed on file  18

 Time for MPS input                             0.00 seconds
 Time for solving problem                       4.72 seconds
 Time for solution output                       0.00 seconds
 Time for constraint functions                  4.70 seconds
 Time for objective function                    0.00 seconds
